client/statsd.go

```diff
@@ -163,3 +168,8 @@ func (c *StatsD) SetHTTPRequestSection(section string) Client {
 func (c *StatsD) ResetHTTPRequestSection() Client {
 	return c.SetHTTPRequestSection(bucket.SectionRequest)
 }
+
+// Handler returns metrics endpoint for prometheus backend
+func (c *StatsD) Handler() http.Handler {
+	return nil
```

Comment:
we'd better return handler always - in this case handler that, for example, returns `405` status code